### DISCUSSION TITLE
Welcome to Hitrava Discussions!

### DISCUSSION THREAD

**Discussion by CTHRU** [CTHRU/Huawei-TCX-Converter#42] [General]
## 👋 Welcome Hitrava Enthusiasts,
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

  To declutter the issues, existing discussions that were started there have been moved to here.

**Comment by jm-my** [comment]
Hi,
thx a lot for your tool.
I wondered if it must be 'the next steps' in the 2nd grey bullitpoint of [https://github.com/CTHRU/Hitrava#step-3---convert-the-data-with-hitrava](url) and the sentense: 'This will convert all available activities in the ZIP file from the previous step.'
